AntWars/AIs/Converter/Classes/AIAnt.cs

```diff
@@ -47,5 +47,47 @@ public bool moveDown()
             return board.BoardObjects.move(ant, newCoords);
         }
 
+        /// <summary>
+        /// Zucker aufnehmen. Die Ameise muss auf dem Zucker stehen.
+        /// </summary>
+        /// <returns></returns>
+        public bool pickUpSugar()
+        {
+            Sugar sugar;
+            if (this.board.BoardObjects.getSugar(this.Coords, out sugar) && ant.Inventory > 0)
+            {
+                int temp = sugar.amount;
+                if (sugar.amount - ant.Inventory <= 0)
+                {
+                    // Zucker bei 0 entfernen
+                    sugar.amount = 0;
+                    this.board.BoardObjects.remove(sugar);
+                }
+                else
+                    sugar.amount = sugar.amount - ant.Inventory;
+
+                ant.Inventory -= (temp - sugar.amount);
+                return true;
+            }
+            else
+                return false;
+        }
+
+        /// <summary>
+        /// Zucker bei der Base abgeben. Die Ameise muss auf der Base stehen.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Wenn du schon c# doc schreibst solltest du die möglichen Werte auch mit in das returns schreiben und bei was sie auftreten.